dashboard/app/controllers/reviewable_projects_controller.rb

```diff
@@ -0,0 +1,39 @@
+class ReviewableProjectsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :decrypt_channel_id, only: [:create]
+
+  check_authorization
+  load_and_authorize_resource :reviewable_project, only: [:destroy]
+
+  # POST /reviewable_projects
+  def create
+    authorize! :create, ReviewableProject
```

Comment:
is there a reason to put this here instead of putting `load_and_authorize_resource` at the beginning without specifying specific methods (which will authorize every method)?

Reply:
Yeah, my understanding here was that I didn't want to `load_resource` in `create` since I would need to first extract the right parameters (i.e. storage_app_id from channel_id) and create the record. Is there a way to make that happen automatically though and leverage `load_and_authorize_resource`

Reply:
ah right makes sense. I'm not sure if there is, so disregard my comment